utils/invoice-actions.js

```diff
@@ -1,40 +1,62 @@
 import { ACCOUNT_NAMES } from './constants'
 import { waitLoad } from '.'
 
+const INVOICE_ACCOUNTS = [
+  ACCOUNT_NAMES.INVOICE,
+  ACCOUNT_NAMES.GEOLOCATION_INVOICE,
+]
+
 export function goToInvoiceAddress(account) {
-  if (account !== ACCOUNT_NAMES.INVOICE) {
+  if (!INVOICE_ACCOUNTS.includes(account)) {
     return
   }
 
   waitLoad()
   cy.waitAndGet('.vtex-omnishipping-1-x-btnDelivery', 1000).click()
-  cy.focused().blur()
+
+  if (account === ACCOUNT_NAMES.INVOICE) {
```

Comment:
why this condition? (might warrant a lil comment)

Reply:
I don't know what this does, actually, I'll try to check the other invoice tests to see

Reply:
apparently this was so the click on the "go to payment" button worked correctly. It appears that it doesn't click properly if another element is already focused so I changed the function that clicks on the button to focus on it before attempting to click which should solve this